### ISSUE TITLE
Text contrast must be above 1.7

### ISSUE THREAD

**Issue by Otterfan** [BCLibraries/bc-libraries-site#33]
WCAG 2.0 AAA requires that text and background have a contrast of at least 7:1 (see [G17: Ensuring that a contrast ratio of at least 7:1 exists between text (and images of text) and background behind the text](http://www.w3.org/TR/2014/NOTE-WCAG20-TECHS-20140916/G17)).

Some of our search result and main page text comes in at 4.7:1, and some even fails the WCAG 2.0 AA standard of 3:1.

There's a contrast checker at: http://webaim.org/resources/contrastchecker/
